src/main/cluster-manager.ts

```diff
@@ -19,8 +28,18 @@ export class ClusterManager extends Singleton {
           cluster.init(port);
         }
       });
+
     }, { fireImmediately: true });
 
+    reaction(() => toJS(clusterStore.enabledClustersList, { recurseEverything: true }), () => {
+      this.updateCatalogSource(clusterStore.enabledClustersList);
```

Comment:
Shouldn't we be removing the concept of enabled clusters anyway?
```suggestion
      this.updateCatalogSource(clusterStore.clustersList);
```

Reply:
Out of scope for this PR but yes I would like to get rid of that.